up.at[row, "pre_n_date"]
报错：
Exception ignored in: <generator object _trim_front.<locals>.<genexpr> at 0x00000287C928DC78>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\core\indexes\base.py", line 5781, in <genexpr>
    while len(strings) > 0 and all(x[0] == " " for x in trimmed):
SystemError: error return without exception set
Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287D1B081B0>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
Exception ignored in: <generator object _trim_front.<locals>.<genexpr> at 0x00000287C928DED0>
Exception ignored in: Traceback (most recent call last):
<generator object _trim_front.<locals>.<genexpr> at 0x00000287C928DD68>  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\core\indexes\base.py", line 5781, in <genexpr>

Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\core\indexes\base.py", line 5781, in <genexpr>
    and all(x.endswith("0") for x in finite)
SystemError: error return without exception set
Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DC78>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
    while len(strings) > 0 and all(x[0] == " " for x in trimmed):
SystemError: error return without exception set
    while len(strings) > 0 and all(x[0] == " " for x in trimmed):
SystemError: error return without exception set    
and all(x.endswith("0") for x in finite)
SystemError: error return without exception setException ignored in: 
<generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DD68>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
Exception ignored in: Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DC78><generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DED0>

Traceback (most recent call last):
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
    and all(x.endswith("0") for x in finite)
SystemError:     error return without exception set
and all(x.endswith("0") for x in finite)
    SystemErrorand all(x.endswith("0") for x in finite)
: SystemErrorerror return without exception set: 
error return without exception set
Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DC78>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DED0>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
    and all(x.endswith("0") for x in finite)
SystemError: error return without exception set
    and all(x.endswith("0") for x in finite)Exception ignored in: 
<generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DC78>SystemError
: Traceback (most recent call last):
error return without exception set  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>

Exception ignored in: <generator object _trim_zeros_float.<locals>._cond.<locals>.<genexpr> at 0x00000287C928DED0>
Traceback (most recent call last):
  File "C:\Users\wqsha\AppData\Roaming\Python\Python37\site-packages\pandas\io\formats\format.py", line 1613, in <genexpr>
    and all(x.endswith("0") for x in finite)
SystemError: error return without exception set
    and all(x.endswith("0") for x in finite)
SystemError: error return without exception set
